api/SWR/suppliers.js

```diff
@@ -1,11 +1,12 @@
 import { useFetchWithParams } from './useFetchWithParams';
+import useGetData from './useGetData';
 
-export const useSuppliers = ({ params, supplierId, shouldFetch }) => {
-  const supplierIdUrl = supplierId ? `/${supplierId}` : '';
-
-  return useFetchWithParams({
+export const useSuppliers = ({ params, shouldFetch }) =>
+  useFetchWithParams({
     params,
     shouldFetch,
-    url: `/suppliers${supplierIdUrl}`
```

Comment:
why have we changed this?

Reply:
i create separate function for request without params